source/manual/emergency-publishing.html.md

```diff
@@ -188,13 +190,31 @@ Follow the instructions above to
 - [Purge the caches and test again](#purge-origin-cache)
 - [Unset your environment variable](#unset-env-var)
 
+> HELP, the banner won't go away. Try out some handy [hints and tips](#help-the-banner-wont-show-no-matter-what)
+
 ---
 ## Troubleshooting
 
 ### Background
 
 The information for the emergency banner is stored in Redis. [Static](https://github.com/alphagov/static) is responsible for displaying the data and we use Jenkins to run [rake tasks in static](https://github.com/alphagov/static/blob/master/lib/tasks/emergency_banner.rake) to set or delete  the appropriate hash in Redis.
 
+<a name="help-the-banner-wont-show-no-matter-what"></a>
```

Comment:
Do we need this? I thought headings automatically turned into anchors.

Reply:
good point - fixed in 633a5d9